src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  const listEmployees = document.querySelector('ul');
+  const employees = document.querySelectorAll(`[${list}]`);
+
+  const sortedEmployees = [...employees].sort((wage1, wage2) => {
+    return getSalary(wage2.getAttribute(list))
+      - getSalary(wage1.getAttribute(list));
+  });
+
+  return sortedEmployees.forEach(person => {
+    listEmployees.appendChild(person);
+  });
+}
+
+function getEmployees(list) {
+  const employees = document.querySelectorAll(`${list}`);
+
+  return [...employees].map(person => {
```

Comment:
```suggestion
  return employees.map(person => {
```
looks like redundant to make new array here. map returns new array and don't mutate current array